assets/src/sass/utilities/_theme-supports.scss

```diff
@@ -2,21 +2,43 @@
+	base-0          $wmui-color-base0           $wmui-color-base100 $wmui-color-accent50,
+	base-10         $wmui-color-base10          $wmui-color-base100 $wmui-color-accent50,
+	base-20         $wmui-color-base20          $wmui-color-base100 $wmui-color-blue90,
+	base-30         $wmui-color-base30          $wmui-color-base0   $wmui-color-base10,
+	base-50         $wmui-color-base50          $wmui-color-base0   $wmui-color-blue50,
+	base-70         $wmui-color-base70          $wmui-color-base0   $wmui-color-blue50,
+	base-80         $wmui-color-base80          $wmui-color-base0   $wmui-color-blue50,
+	base-90         $wmui-color-base90          $wmui-color-base0   $wmui-color-blue50,
+	base-100        $wmui-color-base100         $wmui-color-base10  $wmui-color-blue50,
+	blue-50         $wmui-color-blue50          $wmui-color-base100 $wmui-color-blue90,
+	blue-90         $wmui-color-blue90          $wmui-color-base0   $wmui-color-blue50,
+	blue70          $wmui-color-blue70          $wmui-color-base0   $wmui-color-base50,
+	blue50          $wmui-color-blue50          $wmui-color-base100 $wmui-color-base0,
+	bright-blue     $wmui-color-bright-blue     $wmui-color-base100 $wmui-color-base0,
+	bright-blue70   $wmui-color-bright-blue70   $wmui-color-base0   $wmui-color-base50,
+	bright-green    $wmui-color-bright-green    $wmui-color-base0   $wmui-color-base0,
+	bright-green70  $wmui-color-bright-green70  $wmui-color-base0   $wmui-color-base50,
+	dark-green      $wmui-color-dark-green      $wmui-color-base0   $wmui-color-base0,
+	dark-green70    $wmui-color-dark-green70    $wmui-color-base0   $wmui-color-base50,
+	green         $wmui-color-green           $wmui-color-base100 $wmui-color-base0,
+	green70         $wmui-color-green70         $wmui-color-base0   $wmui-color-base50,
+	orange        $wmui-color-orange          $wmui-color-base100 $wmui-color-base0,
+	orange70        $wmui-color-orange70        $wmui-color-base0   $wmui-color-base50,
+	pink          $wmui-color-pink            $wmui-color-base100 $wmui-color-base0,
+	pink70          $wmui-color-pink70          $wmui-color-base0   $wmui-color-base50,
+	purple        $wmui-color-purple          $wmui-color-base100 $wmui-color-base0,
+	purple70        $wmui-color-purple70        $wmui-color-base0   $wmui-color-base50,
+	red           $wmui-color-red             $wmui-color-base100 $wmui-color-base0,
+	red70           $wmui-color-red70           $wmui-color-base0   $wmui-color-red50,
+	red-50          $wmui-color-red50           $wmui-color-base100 $wmui-color-accent90,
+	red-90          $wmui-color-red90           $wmui-color-base0   $wmui-color-blue50,
+	bright-yellow   $wmui-color-bright-yellow   $wmui-color-base0   $wmui-color-blue50,
+	bright-yellow70 $wmui-color-bright-yellow70 $wmui-color-base0   $wmui-color-blue50,
+	yellow        $wmui-color-yellow          $wmui-color-base0   $wmui-color-blue50,
+	yellow70        $wmui-color-yellow70        $wmui-color-base0   $wmui-color-blue50,
```

Comment:
![image](https://user-images.githubusercontent.com/442115/216669729-c871571d-7f38-4ce2-9c33-dbc45dfae091.png)
This is because VS Code adds swatches to recognized colors, so these appear aligned in the editor 🤯